src/libvfcinstrument/libVFCInstrument.cpp

```diff
@@ -52,8 +60,25 @@ namespace {
     struct VfclibInst : public ModulePass {
         static char ID;
         StructType * mca_interface_type;
-
-        VfclibInst() : ModulePass(ID) {}
+        std::set<std::string> SelectedFunctionSet;
+
+        VfclibInst() : ModulePass(ID) {
+	  if (not VfclibInstFunctionFile.empty()) {
+	    std::string line;
+	    std::ifstream loopstream (VfclibInstFunctionFile.c_str());
```

Comment:
Add a small comment here explaining what is going on; such as "Parsing command line arguments to known which functions must be instrumented. The chosen functions are collected in SelectedFunctionSet."